runtime/vanilla/src/lib.rs

```diff
@@ -396,7 +441,9 @@ construct_runtime!(
         Loans: pallet_loans::{Pallet, Call, Storage, Event<T>, Config},
         Staking: pallet_staking::{Pallet, Call, Storage, Event<T>, Config},
         OcwOracle: pallet_ocw_oracle::{Pallet, Call, Storage, Event<T>, ValidateUnsigned},
+        VanillaOracle: orml_oracle::<Instance1>::{Pallet, Storage, Call, Config<T>, Event<T>},
```

Comment:
do we need two oracles?

OcwOracle, VanillaOracle

Reply:
Ok, will remove OcwOracle later